about.html

```diff
@@ -0,0 +1,110 @@
+            <ul id="menuList">
+                <li><a href="./index.html">Home</a></li>
+                <li><a href="#logoDiv">Logo</a></li>
+                <li><a href="#history">History</a></li>
+                <li><a href="#sponser">Sponsers</a></li>
+            </ul>
+            <ul id="deskList">
+                <li><a href="./about.html" id="aBtn">About</a></li>
+                <li><a href="#logoDiv" class="bBtn">Logo</a></li>
+                <li><a href="#history" class="bBtn">History</a></li>
+                <li><a href="#sponser" class="bBtn">Sponsers</a></li>
+                <li><button type="button" id="gamefairBtn">Campaign</button></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section id="about">
+            <h2 id="hello" class="aboutDesk">"Hello! Sharing world!"</h2>
+            <h1 id="aboutGamecon">GAMING TRADE FAIR,<br>GAMECON 2023</h1>
+            <div id="aboutDetail">
+                <p>Gamecon is a video game conference similar to E3, and in the past was a large convention where
+                    thousands of players and press could flock to witness the latest developments in video games. <span
+                        class="desbr">Alongside Gamecon is Devcon, a portion of the convention dedicated to video game
+                        development, though activities of Devcon continue year-round</span></p>
+            </div>
+            <p id="contact">Please contact us per Email for any further questions about Gamecon 2023<br><a
+                    href="https://www.gamimgfair2023@gamecon.org" type="Email">gamimgfair2023@gamecon.org</a></p>
+        </section>
+
+        <section id="logoDiv">
+            <h2 class="titles">Gaming Trade Fair,<br class="mobbr">Gamecon 2023 Logo</h2>
+            <i class="aboutdash"></i>
+            <p id="logoDesc">The logo of Gamecon 2023 was decided through the logo competition <span class="desbr">After
+                    online public voting and Selection committee's judging process, the logo from Soban Syeed won the
+                    prize. After consultation with Mr. Syeed, we optimized the prize-winning logo for this year's
+                    Gamecon as following:</span></p>
+            <div id="logoBox">
+                <img src="./images/Gamecon2023.png" alt="Gamecon Logo" id="logo">
+            </div>
+            <i class="hrGrey"></i>
+        </section>
+
+        <section id="history">
+            <h2 class="titles">See the previous Gamecon</h2>
+            <i class="aboutdash"></i>
+            <p id="historyDesc">Take a look at last two Gamecons</p>
+            <div id="oldConContainer">
+                <div class="oldGamecon" id="g22">
+                    <h2 class="previousYear">2022<br>
+                        <p class="previousDetail">Gamecon 2022 in Germany</p>
+                    </h2>
+                </div>
```

Comment:
- [ ] For best practices and semantic structure, a header tag should not have any children of its own, kindly remove the `p` tag from inside.

Reply:
Thanks. I have made the changes
![Screenshot 2023-06-30 013615](https://github.com/SnakyMz/Gamecon/assets/115364362/fbf26463-373a-44e3-8cbd-8f9065d25c0b)